src/Sulu/Bundle/MediaBundle/Resources/js/views/MediaOverview/MediaOverview.js

```diff
@@ -132,11 +178,20 @@ export default withToolbar(MediaOverview, function() {
 
     return {
         locale,
-        disableAll: this.collectionStore.loading,
-        backButton: (this.collectionId !== undefined)
+        disableAll: loading,
+        backButton: (this.collectionId)
             ? {
                 onClick: () => {
-                    router.navigate(COLLECTION_ROUTE, {id: this.parentId}, {locale: this.locale.get(), page: 1});
+                    router.navigate(
+                        COLLECTION_ROUTE,
+                        {
+                            id: this.collectionStore.parentId,
+                        },
+                        {
```

Comment:
That's probably not working as intended, because of the latest refactoring of the router. Just merge these two objects in here.